pkg/crc/machine/start.go

```diff
@@ -447,6 +446,19 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 		}
 	}
 
+	// Update the user cert to 'admin-kubeconfig-client-ca' configmap
+	userCA, err := userCA(constants.KubeconfigFilePath)
```

Comment:
I'd prefer that we don't use the `user` terminology at all in this commit, this is all about changing the kubeconfig `admin` client CA both in the kubeconfig file and in the cluster. Nothing related to `user`.

Reply:
`admin` is a user which clientCA need to updated so I can change it to `userClientCA` ?

Reply:
why the indirection? The CA will be used for the admin user, so let's call it `adminClientCA`? This is close to the name of the configmap: `admin-kubeconfig-client-ca`. Just `clientCA` would also work for me